app/components/concept_details_component/concept_details_component.html.haml

```diff
@@ -5,7 +5,8 @@
   %div.raw-table
     - if @bottom_keys.present?
       = render DropdownContainerComponent.new(title: 'Raw data', id: "accordion-#{@id}") do
-        - top_set, leftover_set, bottom_set = filter_properties(@top_keys, @bottom_keys, @exclude_keys, @concept_properties)
+        - @modified_concept_properties = modify_preflabel_key(@concept_properties)
```

Comment:
no need to have a variable `@modified_concept_properties`

Reply:
just call it in the method `filter_properties` argument

Reply:
done